css/site.styl

```diff
@@ -309,6 +309,19 @@ body.single-programacion
 .home-item-image-caption
   opacity: 0
   transition: opacity $fast-animation-speed
+  
+.home-plus
+  pointer-events: none
+  a 
+    pointer-events: auto
+    svg
+      height: 50px
+  &.only-desktop
+    position: fixed
+    text-align: right
+    top: 60%
```

Comment:
I think its too close to the center. to me, from the design, it seem to be positioned at ~`bottom: 18%`